tests/test_integration/stencil_definitions.py

```diff
@@ -273,3 +273,12 @@ def multibranch_param_conditional(
             out_field = in_field - in_field[1, 0, 0]
         else:
             out_field = in_field
+
+
+@register(externals={"DO_SOMETHING": False})
+def allow_empty_computation(in_field: gtscript.Field[float], out_field: gtscript.Field[float]):
+    with computation(FORWARD), interval(...):
+        out_field = in_field
+    with computation(PARALLEL), interval(...):
+        if __INLINED(DO_SOMETHING):
+            out_field = abs(in_field)
```

Comment:
I wasn't sure what kind of test would be best here. I would prefer a more targeted test for the InitInfoPass, but there may be other parts of the toolchain that introduce asserts for empty computations, or the code-gen may not work. This is heavy-handed, but will guarantee it.

Reply:
In principle having tests targeting only parts of the toolchain, e.g. unit tests, is desired. However the effort and maintenance cost should be justified, so that when you make breaking changes to the toolchain the tests are actually useful during development. I think this is not the case here, so an end-to-end test is fine.